src/Core/MainLoop.js

```diff
@@ -68,7 +68,18 @@ function updateElements(context, geometryLayer, elements) {
         // update attached layers
         for (const attachedLayer of geometryLayer._attachedLayers) {
             if (attachedLayer.ready) {
-                attachedLayer.update(context, attachedLayer, element);
+                const sub = geometryLayer.getObjectToUpdateForAttachedLayers(element);
+
+                if (sub) {
+                    if (__DEBUG__) {
+                        if (!(sub.element.isObject3D && sub.element.material)) {
```

Comment:
Some objects still go through this condition, for example the `Scene` of iTowns. Thus on the `3dtiles` example I can't navigate and move at all.

Reply:
It may actually not really a bug.

Do we really want to pass all the root of 3dtiles scene to attached layer? (in which case the attached layer needs to be aware that it may receive a hierarchy, rather than a single object). Or do we want to pass the drawable elements? (in this case the code in getObjectToUpdateForAttachedLayers should collect all objects in `content` that have a material).

I think I prefer the second option. WDYT?

Reply:
Since I'm not sure about this, I removed the `sub.element.material` check for now.

Reply:
I'll go with the second option as well.

Reply:
I implemented it. Note that I had to adapt 3dTilesDebug to reflect this change - this part is a bit tedious due to the way we handle gltfUpAxis (= we rotate the matrix of the model but the boundingVolume must not be affected by this rotation).